index.bs

```diff
@@ -85,36 +83,37 @@ Issue: We should consider having further normative restrictions on file names th
 never be allowed using this API, rather than leaving it entirely up to underlying file
 systems.
 
-A <dfn lt="file|file entry">file entry</dfn> additionally consists of
-<dfn for="file entry">binary data</dfn> (a [=byte sequence=]) and a
+A <dfn export id=file>file entry</dfn> additionally consists of
+<dfn for="file entry" export>binary data</dfn> (a [=byte sequence=]) and a
 <dfn for="file entry">modification timestamp</dfn> (a number representing the number of milliseconds since the <a spec=FileAPI>Unix Epoch</a>).
 
-A <dfn lt="directory|directory entry">directory entry</dfn> additionally consists of a [=/set=] of
-<dfn for="directory entry">children</dfn>, which are themselves [=/entries=]. Each member is either a [=/file=] or a [=directory=].
+A <dfn export id=directory>directory entry</dfn> additionally consists of a [=/set=] of
+<dfn for="directory entry">children</dfn>, which are themselves [=/file system entries=].
```

Comment:
And this?

Reply:
Nope, I don't think so. It's not used in the PR I just put up to remove duplication, at least https://github.com/WICG/file-system-access/pull/392/